Text/AbbyyOCR/README.md

```diff
@@ -0,0 +1,435 @@
+
+Open PowerSkills.sln in the root directory of this Power Skills directory in Visual Studio.  
+
+In the Solution Explorer, Right Click on Text and Choose Add -> Existing Project.
+
+Locate and choose AbbyyOCR.csproj within the \Text\ABBYOCR directory and add it.  
+
+Open AbbyyOCR.cs and update the following lines:
+
+```code
+        private const string ApplicationId = @"[Enter ABBYY Application ID]";
+        private const string Password = @"[Enter ABBYY OCR Password]";
+        private const string ServiceUrl = "[Enter ABBYY Service URL such as https://cloud-westus.ocrsdk.com]";
+```
+
+This code assumes that the images will be coming in, in either English, Arabic or Hebrew.  If you wish to change this, update the below code in the ProcessImageAsync function. You can find the list of [supported languages here](https://www.ocrsdk.com/documentation/specifications/recognition-languages/).
+
+```code
+var imageParams = new ImageProcessingParams
+{
+     ExportFormats = new[] { ExportFormat.Docx, ExportFormat.Txt, },
+     Language = "English,Arabic,Hebrew",
+};
+```
+
+
+# Test
+
+At this point we can test this function.  Click on ABBYYOCR in the Solution Explorer and choose F5.  Once it is running you will see a URL such as http://localhost:7071/api/AbbyyOCR.
+
+Open Postman and create a POST request to this URL.
+
+In the Body of the request, change the type to raw and enter JSON in a format such as this the below JSON.
+
+NOTE: For your test, you will need to not only have the URL references to the Blob image which is placed in the formURL field, but also create a SaS Token for this file which is located in the formSasToken field.
+
+```json
+{
+  "values": 
+  [
+     {
+      "recordId": "foo1",
+      "data": { 
+             "formUrl":  "https://azsdemos.blob.core.windows.net/blob-files/image1.JPG",
+             "formSasToken":  "?sv=..."
+      }
+    },
+         {
+      "recordId": "foo2",
+      "data": { 
+             "formUrl":  "https://azsdemos.blob.core.windows.net/blob-files/image2.JPG",
+             "formSasToken":  "?sv=..."
+      }
+    }
+  ]
+}
+```
+
+The output will look similar to this:
+
+```json
+{
+    "values": [
+        {
+            "recordId": "foo2",
+            "data": {
+                "content": "التنمر..."
+            },
+            "errors": [],
+            "warnings": []
+        },
+        {
+                ...
+        }
+    ]
+}
+```
+
+# Deploy
+
+At this point you can deploy this code to Azure Functions by right clicking on the ABBYYOCR project in the Solution Explorer and choosing Publish. Deploy this to Azure Functions (App Services).
+
+Once deployed, test again.   You will likely need to open this function in the Azure Portal to get the full URL which will include a code for authentication such as https://my-test-abbyy.azurewebsites.net/api/AbbyyOCR?code=/gECNMreXdczJhGZKhziNiMnABC123==
+
+Test this once again in Postman before deploying to Azure Cognitive Search.
```

Comment:
Can you also add a deployment button like the one we have on other skills?

Reply:
Not sure how to do that.  I will add this as a later followup to learn how to do this.

Reply:
Every skill in this repo has one, should be no trouble copying over. You need to copy the code from [another skill's readme](https://raw.githubusercontent.com/Azure-Samples/azure-search-power-skills/master/Template/HelloWorld/README.md), changing the path in the markdown: `[![Deploy to Azure](https://azuredeploy.net/deploybutton.svg)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure-Samples%2Fazure-search-power-skills%2Fmaster%2FTemplate%2FHelloWorld%2Fazuredeploy.json)`. Then you need to copy over the [azuredeploy.json](https://github.com/Azure-Samples/azure-search-power-skills/blob/master/Template/HelloWorld/azuredeploy.json) file, changing a few paths and strings in there that should be fairly easy to locate. LMK if you need any more help with this.